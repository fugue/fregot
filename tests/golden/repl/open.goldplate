{
    "command": "fregot",
    "arguments": ["repl", "--no-history-file"],
    "asserts": [
        {"exit_code": 0},
        {"stderr": "${GOLDPLATE_NAME}.stderr"},
        {"stdout": "${GOLDPLATE_NAME}.stdout"}
    ],
    "stdin": [
        ":open mypkg1",
        "r = 1",
        "r",
        ":open data.mypkg2",
        "r = 2",
        "r",
        ":open data.mypkg1",
        "r",
        "data.mypkg2.r"
    ]
}
