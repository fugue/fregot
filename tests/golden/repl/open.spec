{
    "command": "fregot",
    "arguments": ["repl"],
    "asserts": [
        {"exit_code": 0},
        {"stderr": "${SPEC_NAME}.stderr"},
        {"stdout": "${SPEC_NAME}.stdout"}
    ],
    "stdin": [
        ":open mypkg1",
        "r = 1",
        "r",
        ":open data.mypkg2",
        "r = 2",
        "r",
        ":open data.mypkg1",
        "r",
        "data.mypkg2.r"
    ]
}
