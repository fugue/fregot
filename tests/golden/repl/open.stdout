repl% mypkg1% mypkg1% [36m=[0m [31m1[0m
mypkg1% mypkg2% mypkg2% [36m=[0m [31m2[0m
mypkg2% mypkg1% [36m=[0m [31m1[0m
mypkg1% [36m=[0m [31m2[0m
mypkg1% 